### ISSUE TITLE
Onboard, missing dependency

### ISSUE THREAD

**Issue by fusion809** [shiznix/unity-gentoo#117]
Hi,

I've noticed that the `app-accessibility/onboard` package provided by this overlay is missing a dependency, namely `dev-python/python-distutils-extra`. 

Thanks for your time,
Brenton

**Comment by shiznix** [comment]
Fixed in commmit 881d125ee3a607ef8748716c5d09a11b5439b255

Thanks :)
